bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/bridge-types.xml

```diff
@@ -8,6 +8,7 @@
 	<bridge-type id="weather-api">
 		<label>OpenWeatherMap Account</label>
 		<description>Provides access to the OpenWeatherMap API.</description>
+		<semantic-equipment-tag>WebService</semantic-equipment-tag>
```

Comment:
@andrewfg : is there a logic to use WebService for the bridge (account) and WeatherService for the things?
Shouldn't we have WeatherService here also?

Reply:
Depends, if the bridge is just an account / proxy for other child Things, Webservice is fine.
If the bridge has channels and provides weather information by itself, it can be tagged WeatherService.